src/TabComplete.js

```diff
@@ -45,21 +48,32 @@ class TabComplete {
         this.isFirstWord = false; // true if you tab-complete on the first word
         this.enterTabCompleteTimerId = null;
         this.inPassiveMode = false;
+        this.memberTabOrder = {};
```

Comment:
What is this? A map? What are the keys? Integers to represent the ordering? User IDs? What are the values? Comment please!